Improbable/Stdlib/Improbable.Stdlib/WhenAllComponents.cs

```diff
@@ -55,7 +55,7 @@ private void Added(AddComponentOp op)
                 return;
             }
 
-            if (!counts.TryGetValue(op.EntityId, out var count))
+            if (!counts.TryGetValue(op.EntityId, out _))
```

Comment:
```suggestion
            if (!counts.Contains(op.EntityId))
```

Reply:
This, or revert this change and set `count = 0;` inside the if, and counts.Add that. Then use `count + 1` over  `counts[op.EntityId] + 1` below.